src/path_manager_example.cpp

```diff
@@ -400,14 +438,14 @@ void path_manager_example::dubinsParameters(const waypoint_s start_node, const w
         float theta, theta2;
         // compute L1
         theta = atan2f(cre(1) - crs(1), cre(0) - crs(0));
-        float L1 = (crs - cre).size() + R*mo(2*M_PI_F + mo(theta - M_PI_2_F) - mo(_dubinspath.chis - M_PI_2_F))
+        float L1 = (crs - cre).norm() + R*mo(2*M_PI_F + mo(theta - M_PI_2_F) - mo(_dubinspath.chis - M_PI_2_F))
                 + R*mo(2*M_PI_F + mo(_dubinspath.chie - M_PI_2_F) - mo(theta - M_PI_2_F));
 
         // compute L2
-        ell = (cle - crs).size();
+        ell = (cle - crs).norm();
         theta = atan2f(cle(1) - crs(1), cle(0) - crs(0));
         float L2;
-        if(2*R/ell > 1.0f || 2*R/ell < -1.0f)
+        if(2*R > ell)
```

Comment:
It has been tested. I don't know why the other statement is necessary since R and ell are always positive.